api/app/googleflood.py

```diff
@@ -0,0 +1,106 @@
+"""Get data from Google Floods API"""
+
+import logging
+from os import getenv
+from urllib.parse import urlencode
+
+import requests
+from fastapi import HTTPException
+
+logger = logging.getLogger(__name__)
+
+GOOGLE_FLOODS_API_KEY = getenv("GOOGLE_FLOODS_API_KEY", "")
+if GOOGLE_FLOODS_API_KEY == "":
+    logger.warning("Missing backend parameter: GOOGLE_FLOODS_API_KEY")
+
+
+def format_gauge_to_geojson(data):
+    """Format Gauge data to GeoJSON"""
+    geojson = {
+        "type": "Feature",
+        "geometry": {
+            "type": "Point",
+            "coordinates": [
+                data["gaugeLocation"]["longitude"],
+                data["gaugeLocation"]["latitude"],
+            ],
+        },
+        "properties": {
+            "gaugeId": data["gaugeId"],
+            "issuedTime": data["issuedTime"],
+            "siteName": data["siteName"],
+            "river": (
+                data["river"] if "river" in data and len(data["river"]) > 1 else None
+            ),
```

Comment:
"river" can come in as a string with just 1 space (`" "`). This ensures there's a real value